packages/common/src/authentication/api/hooks/useUserDetails.ts

```diff
@@ -4,9 +4,12 @@ import { useAuthApi } from './useAuthApi';
 
 export const useUserDetails = () => {
   const api = useAuthApi();
-  return useMutation<UserNode, unknown, string | undefined, unknown>(
-    api.get.me
-  );
+  return useMutation<
+    UserNode | undefined,
```

Comment:
required because of the error suppression which isn't returning an object when there's a problem